### PR TITLE
feature/transaction details

### PR DESCRIPTION
**Pull request by benjaminpfleghaar**

### CODE REVIEW

File: app/not-found.tsx
Comment (StephMode) [inline app/not-found.tsx:1]: Great implementation of https://nextjs.org/docs/app/api-reference/file-conventions/not-found 👍🏻

File: components/layout/Status.tsx
Comment (StephMode) [inline components/layout/Status.tsx:1]: I think it's great how you desinged the "status message" as it's own component.

File: components/ui/GhostButton.tsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 	);
 }
 
-const StyledButton = styled.button`
+const StyledButton = styled.button<{ $color?: string }>`
 	height: var(--spacing-48);
 	font: var(--font-medium-14);
 	padding: 0 var(--spacing-24);
 	border-radius: var(--spacing-8);
+	color: ${(props) => (props.$color === "red" ? "var(--color-red-90)" : "inherit")};
```
Comment (StephMode) [inline components/ui/GhostButton.tsx:16]: This looks like the "Theme" feature in StyledComponents or am I mistaken?

**Review by StephMode — APPROVED** [review]
Great implementation.

Only one question to make sure I have a complete understanding of your code. Apart from that, everything looks very good to me.

File: components/ui/GhostButton.tsx (hunk 1 above)
Comment (benjaminpfleghaar) [reply components/ui/GhostButton.tsx:16]: I don't use the Theme feature but the [passed props](https://styled-components.com/docs/basics#passed-props) functionality to change the color of the button. Default inherits the color from the body, which is white in this case. And for the delete button I pass in "red" to change the color according to the CSS variable.

**Pull request merged**